template_functions.go

```diff
@@ -145,6 +146,35 @@ var (
 		},
 		"slug": Slug,
 		"even": func(a int) bool { return (a % 2) == 0 },
+
+		// Using https://github.com/xeonx/timeago
+		"timeago": func(viewArgs map[string]interface{}, datetime time.Time) string {
```

Comment:
If possible can you modify this so it behaves similar to the i18template function. 
Basically allow for variable number of arguments
- If one argument format the time as english
- if two arguments, and second one is string assume that is the region
- if two arguments, and second is map assume that is the viewArgs and extract the CurrentLocaleViewArg from the map